cartography/intel/gcp/sql.py

```diff
@@ -141,6 +141,63 @@ def get_sql_users(sql: Resource, sql_instances: List[Dict], project_id: str) ->
     return sql_users
 
 
+@timeit
+def get_sql_databases(sql: Resource, sql_instances: List[Dict], project_id: str) -> List[Dict]:
+    """
+        Returns a list of sql instance database for a given project.
+
+        :type sql: Resource
+        :param sql: The sql resource created by googleapiclient.discovery.build()
+
+        :type sql_instances: List
+        :type sql_instances: List of sql instances
+
+        :type project_id: str
+        :param project_id: Current Google Project Id
+
+        :rtype: list
+        :return: List of Sql Instance Database
+    """
+    sql_database = []
+    for inst in sql_instances:
+        try:
+            request = sql.databases().list(project=project_id, instance=f"{inst['name']}")
+            while request is not None:
+                response = request.execute()
+                if response.get('items', []):
+                    for item in response['items']:
+                        item['instance_id'] = inst['id']
+                        item['id'] = f"projects/{project_id}/instances/{inst['name']}/databases/{item['name']}"
+                        item['consolelink'] = gcp_console_link.get_console_link(
```

Comment:
is consolelink implemented for databases if not then leave with empty string now?

Reply:
at the testing time find consolelink